releasenotes/notes/ipa-build-workflow-cde24f3b9e3e0667.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
```

Comment:
Needs a second list item saying that the stackhpc ipa images are enabled by default